server/Entities/Models/User.cs

```diff
@@ -9,12 +9,15 @@ public class User
     public User()
     {
       Gender = Gender.NotSpecified;
+      IsOauth = false;
     }
 
     public int Id { get; set; }
     public string Email { get; set; }
     public string Password { get; set; }
 
+    public bool IsOauth { get; set; }
```

Comment:
зачем это поле7

Reply:
Хотел отделить пользователей которые вошли с гуглом и без гугла, потом мб нужно будет запретить смену почты...

Reply:
i'd better save google_id or somth alike which google should give you when logging in.
but ok, you can keep it in case google doesn't give id with email in auth response